scripts/bootstrap_cluster.sh

```diff
@@ -119,6 +119,29 @@ install_argocd_helm_chart() {
     --set controller.extraArgs[0]='--application-namespaces="*"'
 }
 
+install_istio() {
+    local tenant_name=$1
+    local cluster_token=$2
+    local ip_adress=$(kubectl cluster-info | head -n 1 | grep -oE '[^[:space:]]+' | sed -n '7p')
```

Comment:
can we verify this works for any cluster? @opsved

Reply:
curl is not to check - it is to install istio and create DNS record